src/ls.js

```diff
@@ -86,7 +86,9 @@ function _ls(options, paths) {
         // use glob, because it's simple
         glob.sync(p + globPatternRecursive, { dot: options.all, follow: options.link })
           .forEach(function (item) {
-            pushFile(item, path.relative(p, item));
+            if (path.relative(p, item)) {
```

Comment:
Could you add a comment explaining that this is because the glob pattern returns the directory itself? Otherwise, LGTM